scripts/publish_infra_k8s_outputs.py

```
@@ -0,0 +1,163 @@
+    """Publish outputs for the wildside-infra-k8s action.
+
+    This command reads computed values from environment variables and
+    writes them to GITHUB_OUTPUT for use by subsequent workflow steps.
+    """
+    # Resolve GITHUB_OUTPUT path
+    github_output_raw = resolve_input(
+        github_output,
+        InputResolution(env_key="GITHUB_OUTPUT", as_path=True),
+    )
+    if not github_output_raw:
+        raise SystemExit("GITHUB_OUTPUT or --github-output must be provided")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)